src/main/java/ZadaniaWzorceProjektowe_10_09/Zadanie4/Fifty.java

```diff
@@ -0,0 +1,18 @@
+package ZadaniaWzorceProjektowe_10_09.Zadanie4;
+
+public class Fifty extends WithdrawPower {
+    @Override
+    protected int getDenomination() {
+        return 50;
+    }
+
+    @Override
+    protected int getAllowable() {
+        return BASE * 15;
+    }
+
+    @Override
+    protected String getRole() {
+        return "50 PLN bank notes.";
```

Comment:
Fajnie by było pobrać tutaj wartość z metody getDenomination() i zbudowac stringa, bo jak zmienisz wartość np na 500 to string cały czas 50 będzie wyświetlać :)